src/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -215,6 +215,7 @@ def do_fgs():
             30.0,
         )
         LOGGER.info("kicking off FGS")
+        yield from bps.wait()  # Wait for all moves with no assigned group
```

Comment:
Should: The ordering of the logs here is deceptive. I think we should have two logs, one before the wait that logs basically the same as the comment then we log saying we're kicking off and do the kick off